frontend/ui/src/pages/Channels/ChannelsMainPage/SourceInfo/index.tsx

```diff
@@ -70,7 +71,7 @@ const SourceInfo = (props: SourceInfoProps) => {
           </div>
 
           <div className={styles.channelButton}>
-            <button type="button" className={styles.addChannelButton} onClick={props.onAddChannelClick}>
+            <button type="button" className={styles.addChannelButton} onClick={() => props.onAddChannelClick()}>
```

Comment:
We couldn't re-use the function here because the `onClick` event here is different. @lucapette